src/lib/sap_ha/system/hana.rb

```diff
@@ -290,6 +262,10 @@ def adjust_non_production_system(system_id, options = {})
           global_ini = CFA::GlobalIni.new(global_ini_path)
           global_ini.load
           global_ini.set_config('memorymanager', 'global_allocation_limit', options[:global_alloc_limit])
+          global_ini.set_config('ha_dr_provider_SAPHanaSR', 'provider', 'SAPHanaSR')
```

Comment:
:thinking:  Just a note: This block and the block above look very similar, maybe it would make sense to share them in some method, then you would need to touch only one place if doing changes...